### PR TITLE
refactor(Tabs): move tabs to Elements

### PR DESCRIPTION
**Pull request by CITguy**
JIRA 1: https://jira.rax.io/browse/FRMW-628
JIRA 2: https://jira.rax.io/browse/FRMW-794

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM

### CODE REVIEW

**Comment by CITguy** [comment]
### BEFORE
tabs-before

### AFTER
tabs-after

File: demo/less/_code.less [hunk 1]
```diff
@@ -31,5 +31,6 @@ pre {
     padding: 0;
     margin: 0;
     border-radius: 0;
+    white-space: pre-wrap;
```
Comment (CITguy) [inline demo/less/_code.less]: example source code was causing layout issues

**Comment by glynnis** [comment]
I would also add this to the "don't" bullets (per the description of https://jira.rax.io/browse/FRMW-794):
> * Don't use tabs as navigation with entire pages inside. Rather than putting extensive or lengthy 
> content in a tabbed area, create a new page with an additional link in the left hand nav.

Per the do/don't specs, we shouldn't have any instances where "do" or "don't" examples are used without the other. Can you add a "do" example here that shows a shorter version of the Rackspace Images/Saved Images tabbed option tables from the Create New Cloud Servers page (in code)?
screen shot 2016-05-11 at 10 17 10 am

Suggested text for "do" example:
> * Do use tabs for complex forms that use option tables. Tabs are a good way to display "either/or" 
> choices to the user.
> * Do use tabs to show closely related information cleanly.

**Comment by glynnis** [comment]
This has a "dev LGTM" box checked but it doesn't look like anyone has reviewed this yet. Thought I'd mention it in case it was checked in error.

File: src/utilities/rxBytesConvert/scripts/rxBytesConvert.js [hunk 2]
```diff
@@ -44,4 +44,5 @@ angular.module('encore.ui.utilities')
             return result.toFixed(2) + ' ' + units[index];
         }
     };
-});
\ No newline at end of file
+});
+
```
Comment (CITguy) [inline src/utilities/rxBytesConvert/scripts/rxBytesConvert.js]: Newline added to appease eslint

File: src/elements/Tabs/docs/tabs.docs.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  <li>
+    <code>vertical</code>, <code>pills</code>, and <code>justified</code>
+    tabs are currently unsupported.
+  </li>
+</ul>
+
+<h3>Alternative Lorem Ipsums</h3>
+<rx-example name="tabs.simple"></rx-example>
+
+<div class="example avoid">
```
Comment (parlarjb) [inline src/elements/Tabs/docs/tabs.docs.html]: Could you please change this class to `example-case` per https://github.com/rackerlabs/encore-ui/pull/1654 ?

File: src/utilities/rxBytesConvert/scripts/rxBytesConvert.js (hunk 2 above)
Comment (100stacks) [inline src/utilities/rxBytesConvert/scripts/rxBytesConvert.js]: Looks like you and @glynnis fixed this eslint issue with https://github.com/rackerlabs/encore-ui/pull/1654.

Comment (CITguy) [inline src/utilities/rxBytesConvert/scripts/rxBytesConvert.js]: I'll be rebasing

File: demo/less/demoApp.less [hunk 4]
```diff
@@ -146,7 +146,7 @@ p {
 .example-case {
   border-style: solid;
   border-width: 9px 2px 2px 2px;
-  padding: 0 10px;
+  padding: 0 10px 10px;
```
Comment (CITguy) [inline demo/less/demoApp.less]: ### Before
screen shot 2016-05-12 at 11 30 56 am

### After
screen shot 2016-05-12 at 11 32 01 am

**Comment by parlarjb** [comment]
LGTM

**Comment by 100stacks** [comment]
LGTM

**Comment by CITguy** [comment]
Screenshots updated

File: src/elements/Tabs/docs/tabs.docs.html [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+  <a href="http://angular-ui.github.io/bootstrap/#/tabs">the Angular-UI Bootstrap Tabs plugin</a>,
+  which is included as a dependency for EncoreUI. Usage is the exact same as
+  demoed on the Angular-UI Bootstrap documentation.
+</p>
+
+<h3>Disclaimer</h3>
+<ul class="list">
+  <li>Only the default horizontal tabs are supported by these styles.</li>
+  <li>
+    <code>vertical</code>, <code>pills</code>, and <code>justified</code>
```
Comment (glynnis) [inline src/elements/Tabs/docs/tabs.docs.html]: Not sure I understand what this means. What are vertical, pills, and justified tabs?

File: src/elements/Tabs/docs/tabs.docs.html [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+    tabs are currently unsupported.
+  </li>
+</ul>
+
+<h3>Alternative Lorem Ipsums</h3>
```
Comment (glynnis) [inline src/elements/Tabs/docs/tabs.docs.html]: Can we lose this header? It doesn't make much sense as a title. We may not need a title here at all.

File: src/elements/Tabs/docs/tabs.docs.html (hunk 5 above)
Comment (CITguy) [inline src/elements/Tabs/docs/tabs.docs.html]: @glynnis, this disclaimer was originally in the js docs, but it refers to the tab styles that ngBootstrap has that we do not support (see below).

screen shot 2016-05-12 at 4 22 16 pm

**Comment by glynnis** [comment]
Once the Lorem Ipsum header is out, this LGTM! 🎉

**Comment by CITguy** [comment]
@glynnis, heading removed. Screenshots updated

**Pull request merged**